doc/about.rst

```diff
@@ -96,86 +96,180 @@ following paper:
 Artwork
 -------
 
-High quality PNG and SVG logos are available in the `doc/logos/ <https://github.com/scikit-learn/scikit-learn/tree/master/doc/logos>`_ source directory.
+High quality PNG and SVG logos are available in the `doc/logos/
+<https://github.com/scikit-learn/scikit-learn/tree/master/doc/logos>`_
+source directory.
 
 .. image:: images/scikit-learn-logo-notext.png
    :align: center
 
 Funding
 -------
+Scikit-Learn is a community driven project, however institutional and private
+grants help to assure its sustainability.
 
-`INRIA <https://www.inria.fr>`_ actively supports this project. It has
-provided funding for Fabian Pedregosa (2010-2012), Jaques Grobler
-(2012-2013) and Olivier Grisel (2013-2017) to work on this project
-full-time. It also hosts coding sprints and other events.
+The project would like to thank the following funders. 
 
-.. image:: images/inria-logo.jpg
-   :width: 200pt
+...................................
+
+The `Scikit-Learn Consortium at Inria Foundation
+<https://scikit-learn.fondation-inria.fr/en/home/>`_ `Members 
+<https://scikit-learn.fondation-inria.fr/en/home/#sponsors>`_ fund Olivier
+Grisel, Guillaume Lemaitre, Jérémie Du Boisberranger and Chiara Marmo.
+
+**Platinum Members**
+
+
+.. |msn| image:: images/microsoft.png
+   :width: 100pt
+   :target: https://www.microsoft.com/
+
+.. |bcg| image:: images/bcg.png
+   :width: 100pt
+   :target: https://www.bcg.com/beyond-consulting/bcg-gamma/default.aspx
+
++-------+-------+
```

Comment:
Is it possible to use:

```rst
.. table::
   :class: sk-sponser-table
```

To give the table a custom class, `sk-sponser-table`, so we can remove the blue background and the lines here:

<img width="339" alt="Screen Shot 2019-09-23 at 7 12 00 PM" src="https://user-images.githubusercontent.com/5402633/65469607-0bdace00-de36-11e9-8ce7-35e72a910ee2.png">

Reply:
done in ab7187829971db1cdafdafb0f0846c24cc44cb96